### ISSUE TITLE
Checksum does not match error when adding a attribute

### ISSUE THREAD

**Issue by Matthijz98** [adamghill/django-unicorn#490]
Hi 👋👋👋👋👋

I am trying to make my first project with unicorn and loving it so far.
But i have one problem. A am trying to make a simple list of accounts with search and filter options.
That worked fine but now i want to add some pagination info and i keep getting {"error": "Checksum does not match"}

When i commend out ``self.last_page = int(paginator.num_pages)`` or change it to ``self.last_page = 1`` it works fine
I probaly missing something but i can not find out why this breaks.

My component is:
```python
class AccountSearchView(UnicornView):
    id_search_term = ''
    name_search_term = ''
    description_search_term = ''

    current_page = 1
    next_page = None
    previous_page = None
    first_page = None
    last_page = None

    def go_to_page(self, page_number):
        self.current_page = page_number
        self.accounts()

    def accounts(self):
        print(self.id_search_term)
        accounts = Account.objects.filter(id__icontains=self.id_search_term)\
            .filter(name__icontains=self.name_search_term)

        paginator = Paginator(accounts, 50)

        try:
            page = paginator.page(self.current_page)
        except PageNotAnInteger:
            # if page is not an integer, deliver the first page
            page = paginator.page(self.current_page)
        except EmptyPage:
            # if the page is out of range, deliver the last pa
            page = paginator.page(paginator.num_pages)

        self.last_page = int(paginator.num_pages) <---- as soon i add this line the component breaks

        return page
```

and my template is:

```html
<div>
    <div class="card">
        <div class="card-body" style="padding: 0">
            <table class="table">
                <thead>
                    <tr>
                        <th>User ID</th>
                        <th>Name</th>
                        <th>Description</th>
                    </tr>
                    <tr>
                        <th>
                            <input class="form-control form-control-sm" type="text" unicorn:model="id_search_term"
                                   placeholder="Search" id="id_search_term"></input>
                        </th>
                        <th>
                            <input class="form-control form-control-sm" type="text" unicorn:model="name_search_term"
                                   placeholder="Search" id="name_search_term"></input>
                        </th>
                        <th>
                            <input class="form-control form-control-sm" type="text" unicorn:model="description_search_term"
                                   placeholder="Search" id="description_search_terms"></input>
                        </th>
                    </tr>
                </thead>
                <tbody id="account_table_body">
                {% for account in accounts %}
                    <tr>
                        <td><a href="{% url 'account_details' account.id %}">{{ account.id }}</a></td>
                        <td>{{ account.name }}</td>
                        <td>{{ account.description|default:'-'}}</td>
                    </tr>
                {% endfor %}

                </tbody>
            </table>
        </div>
    </div>

</div>

```

**Comment by adamghill** [comment]
The checksum is basically a way that `Unicorn` makes sure that the data is valid on every request. Are you seeing any errors in the JS or Python console? 

I pushed [a commit](https://github.com/adamghill/django-unicorn/commit/baf51ce095481450c985b476809041bb4edffdea) which is basically what you have above and here is a video of what happens on my local: https://user-images.githubusercontent.com/317045/206069572-8e2e5901-be35-4b4d-9942-36ed0ebf08d9.mp4. I'm not able to replicate it, but I might be missing something.

You can pull this branch and run the example project with `example/manage.py runserver 0:8080` and go to http://localhost:8080/account_search. Let me know if that works for you or you are still seeing the same error.

**Comment by Matthijz98** [comment]
Thanks for the response.
I have forked the example and you are correct it does not give the error, so i went back to the original code.
For some reason i do not get the checksum error now, but it still does not work as expected.
The first time the component completely updates but after that noting happend when i type something in the input field.
The requests are also smaller:
![image](https://user-images.githubusercontent.com/17087153/207027282-d94e8d84-89be-4c5b-9939-a633dc4f1011.png)
Example of the first request:
![image](https://user-images.githubusercontent.com/17087153/207029760-7a2e25ac-3c92-4fcf-b490-e88a0104f810.png)
Example of the later requests:
![image](https://user-images.githubusercontent.com/17087153/207029896-f38c2782-9af3-411e-aa99-42ea6617835f.png)
I will give it some more tries but i think it is a error on my side now. I have no clue why i do not get the checksum error noting changed and why the same code in the example works deference. I wil update the fork with more of my code to see if it wil reproduce the same error.

**Comment by adamghill** [comment]
This is just a hunch, but what is your [`CACHES` setting](https://docs.djangoproject.com/en/4.1/ref/settings/#std-setting-CACHES)?

**Comment by Matthijz98** [comment]
I figured it out (i think).
It is hard to reproduce but after some time I think found a pattern
If i type a char in the search input and then remove it before the request is done sometimes it keep hanging in a state where:
The frontend is a state that it has been before so the checksum is the same
The backend then sends a update which updated the input fields with the input before i hit backspace
And now it is broken and it hangs and does not update whatever i try. But this did not happen every time for some reason.

I fixed it with some debounce and locking the input while loading works perfectly.

Why this happend the first time when I added the pagination I still have no clue.
